exercises/run-length-encoding/example.js

```diff
@@ -0,0 +1,13 @@
+
+export const encode = (plaintext) => {
+  const consecutiveChars = /([\w\s])\1*/g;
+  return plaintext.replace(consecutiveChars,
+      match => (match.length > 1 ? match.length + match[0] : match[0]));
+};
+
+
+export const decode = (cypher) => {
```

Comment:
Did you decide that you want to keep the name `cypher` for this parameter?  Just curious what you are thinking.